lms/urls.py

```diff
@@ -825,6 +825,11 @@
     url(r'^commerce/', include('commerce.urls', namespace='commerce')),
 )
 
+if settings.FEATURES.get('ENABLE_COURSE_GOALS'):
```

Comment:
I would ditch this. Put the feature flag around the code that consumes the API, not the API itself.